src/components/Button.js

```diff
@@ -1,16 +1,25 @@
 import PropTypes from 'prop-types';
 
 const Button = (props) => {
-  const { value } = props;
-  return <button className="button" type="button">{value}</button>;
+  const { value, bgColor, width } = props;
+
+  return (
+    <button className="button" type="button" style={{ backgroundColor: bgColor, width: `${width}%` }}>
```

Comment:
- [ ] [REQUIRED] Please, don't use inline styling as it is a bad practice.

Reply:
Inline styling is removed, using classNames instead.